import-export-cli/cmd/export.go

```diff
@@ -24,21 +24,27 @@ import (
 )
 
 // Export command related usage Info
-const exportCmdLiteral = "export"
-const exportCmdShortDesc = "Export an API Product in an environment"
+const ExportCmdLiteral = "export"
```

Comment:
Is there any specific reason to use "ExportCmdLiteral" instead of "exportCmdLiteral"? Shall we stick into one. I can see we have used constants in both the forms.

Reply:
Yes, there is a reason to make the first letter capital here. 
In Go, a varaible is exported if it begins with a capital letter [1]. In other words we can say that **ExportCmdLiteral** is refered from another package (**deprecated** package) [2], so we need to keep the first letter in capital, otherwise we cannot refer this variable from that package.

[1] https://tour.golang.org/basics/3
[2] https://github.com/wso2/product-apim-tooling/blob/3c64cf186a13a7a71e67993c97516396ad2bf808/import-export-cli/cmd/deprecated/exportAPI.go#L59

Reply:
Thanks for the explanation @wasuradananjith. Got it now, those are used in the the help command outputs in the deprecated package.

Reply:
No worries. Yes, that is the exact reason.